packages/experiments/src/theming/examples/Theming.Customizers.Example.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+
+import { CommandBar } from 'office-ui-fabric-react/lib/CommandBar';
+import { farItems, items, overflowItems } from 'office-ui-fabric-react/lib/components/CommandBar/examples/data';
+import { Customizer } from 'office-ui-fabric-react/lib/Utilities';
+
+import { HorizontalStack, VerticalStack } from '@uifabric/experiments/lib/Stack';
+import { Text } from '@uifabric/experiments/lib/Text';
+// tslint:disable:max-line-length
+import { CollapsibleSectionRecursiveExample } from '@uifabric/experiments/lib/components/CollapsibleSection/examples/CollapsibleSection.Recursive.Example';
+
+import { DialogExample } from './DialogExample';
+import { defaultTheme, neutralTheme, softTheme, strongTheme, regionStyles } from './Themes';
+
+export class ThemingExample extends React.Component<{}, {}> {
+  /**
+   * Render various components using multiple Customizers.
+   */
+  protected _renderCustomizedComponents(): JSX.Element {
+    return (
+      <HorizontalStack gap={10}>
+        <Customizer settings={{ theme: strongTheme }}>
+          <HorizontalStack.Item grow={true} styles={{ root: { width: '33%', maxWidth: '33%' } }}>
+            <VerticalStack styles={regionStyles} padding={5}>
+              <Text weight="bold">Strong Theme</Text>
+              <CollapsibleSectionRecursiveExample />
+            </VerticalStack>
+          </HorizontalStack.Item>
+        </Customizer>
+        <HorizontalStack.Item grow={true} styles={{ root: { height: 'auto' } }}>
+          <VerticalStack grow={true} fillVertical={true}>
+            <Customizer settings={{ theme: softTheme }}>
```

Comment:
This example, while nice to prove that Customizer works, is an antipattern imo.

I don't think we want partners to pass in random themes everywhere. It means that it will become more difficult to manage an overall theme for the app. And if a library does it, it's almost certainly game over for app themeability.

Reply:
Yes, this is intended to be the anti-pattern example. I had thought of removing this entirely so it's not even considered or used.. Is there a better way to do this with multiple themes?